docs/content/developers/writing-style-guide.md

```diff
@@ -0,0 +1,222 @@
+# Writing Style Guide
+
+This page formalizes the writing conventions we aspire to use in the documentation.  
+It’s a living statement of intent and a reference for all contributors.
+
+## Voice and Tone
+
+In the spirit of the [Code of Conduct](../users/code-of-conduct.md), we want to be clear and encouraging for everyone that bothers to read DDEV’s documentation, rewarding the time and attention they choose to give to it.
+
+### Beginner-Friendly, Expert-Compatible
+
+Write so a DDEV beginner can follow your guidance and a DDEV veteran could use the same content as a reference.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| You can run `ddev restart` to apply changes you’ve made to your global `~/.ddev/global_config.yaml` or per-project `.ddev/config.yaml`. | Just `ddev restart` to apply YAML config changes.
+
+### Be Efficient and Direct
+
+Brevity makes for good documentation!
+
+Few read the docs from start to finish like a book, and explanation can be a barrier to learning. Get to the point, avoiding parentheticals and mid-sentence notes that interrupt the main flow.
+
+Omit extraneous explanation or decorative language that doesn’t help the reader. Instructions don’t need to include “please”. Provide some context for anchor links without overloading them to hinder flow.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| Run `ddev start` and launch the site in a browser. | Please run `ddev start`, then launch the site in a browser.
+| Learn more on the [Extending](../users/extend/customization-extendibility.md) page. | (You can also learn more about this and related topics in [Providing Custom Environment Variables to a Container](../users/extend/customization-extendibility.md).)
+
+### Avoid “Just” and “Easy”
+
+Try not to use language that may talk down to the reader. You may intend for “it’s easy” to be reassuring, but it’s a subjective judgment that can convince someone struggling that they’re doing it wrong. Things could instead be “straightforward” if they’re without nuance, “simple” if they don’t involve complex actions or concepts, or “quick” if they involve one or two steps that’d be fast even on someone’s worst day with the slowest-imagineable machine.
```

Comment:
Oddly, the docs check didn't get run, which should have caught this. We'll see if it runs when you commit this suggestion.

Reply:
I got in the habit of using `[skip ci]` so that’s probably my fault. Thanks @mayankguptadotcom!

Reply:
For PRs that are just docs, there's no need to use [skip ci] because most of the tests are skipped when the system discovers it's just docs. I imagine you already know that.

Reply:
I did not, so thank you!